src/core_modules/capture-core/components/WidgetStagesAndEvents/Stages/Stage/Stage.component.js

```diff
@@ -60,16 +57,12 @@ export const StagePlain = ({ stage, events, classes, className, onCreateNew, ...
                 /> : <Button
                     small
                     secondary
+                    icon={<IconAdd24 />}
```

Comment:
Hey,
IMHO the `IconAdd24` is too big for these buttons. How about using the `IconAdd16` instead? 
<img width="327" alt="Screenshot 2022-11-11 at 14 35 58" src="https://user-images.githubusercontent.com/6817073/201352934-90038a3c-e142-4f89-a143-54b5b669405b.png">
Thanks!

Reply:
Agreed, thanks for saying! Committed a fix.